docs/gl_objects/runners.rst

```diff
@@ -29,6 +29,10 @@ Examples
 --------
 
 Use the ``list()`` and ``all()`` methods to list runners.
+``list()`` - Get a list of specific runners available to the user
+``all()``  - Get a list of all runners in the GitLab instance (specific and 
+shared). Access is restricted to users with administrator access.
```

Comment:
Thanks for the PR @lmilbaum! I realized we have some outdated docs here. Would you mind updating this as well?

We no longer recommend the `all()` method, but instead have a separate manager which brings more consistent behavior between the two endpoints. Like this:

```suggestion
Use the ``runners.list()`` and ``runners_all.list()`` methods to list runners.
``runners.list()`` - Get a list of specific runners available to the user
``runners_all.list()``  - Get a list of all runners in the GitLab instance (specific and 
shared). Access is restricted to users with administrator access.
```

We would also need to change the text below, but I can't suggest it here in the PR - like so:
```diff
-* ``specific`` (``all()`` only)
-* ``shared`` (``all()`` only)
+* ``specific`` (``runners_all.list()`` only)
+* ``shared`` (``runners_all.list()`` only)
```

Thanks again for the PR :bow:

Reply:
NP. Pushed a fix.